.travis.yml

```diff
@@ -34,5 +34,14 @@ before_script:
   - ./bin/travisci
   - travis_retry ./sbt ++$TRAVIS_SCALA_VERSION update
 
-script: ./sbt ++$TRAVIS_SCALA_VERSION coverage test && ./sbt ++$TRAVIS_SCALA_VERSION coverageAggregate
+script: "echo no op"
```

Comment:
I think your diff is out of date. We've switched to use codecov.io in the latest code (instead of coveralls). You may want to re-base the branch.